modules/loader.js

```diff
@@ -189,8 +189,7 @@ __private.loadTransactions = function (cb) {
 				try {
 					transaction = library.logic.transaction.objectNormalize(transaction);
 				} catch (e) {
-					library.logger.debug(['Transaction', id].join(' '), e.toString());
-					if (transaction) { library.logger.debug('Transaction', transaction); }
+					library.logger.debug ('Transaction normalization failed [loader]', {id: id, 'err': e.toString (), tx: transaction});
```

Comment:
If we are to mention module, then can you do the following:

```js
library.logger.debug('Transaction normalization failed', {id: id, err: e.toString(), tx: transaction, module: 'loader'});
```

Adding module name as property to logged object.

This would apply to all instances within this PR.